pyproject.toml

```diff
@@ -1,47 +1,48 @@
-
 [build-system]
 requires = ["setuptools>=64", "setuptools-scm>=8", "packaging", "saxonche"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "imas-data-dictionary"
-description ="The Data Dictionary is the implementation of the Data Model of ITER's Integrated Modelling & Analysis Suite (IMAS)"
-authors = [{name = "ITER Organization", email = "imas-support@iter.org"}]
-readme = {file = "README.md", content-type = "text/markdown"}
-license = {file = "LICENSE-LGPL"}
+description = "The Data Dictionary is the implementation of the Data Model of ITER's Integrated Modelling & Analysis Suite (IMAS)"
+authors = [{ name = "ITER Organization", email = "imas-support@iter.org" }]
+readme = { file = "README.md", content-type = "text/markdown" }
+license = { file = "LICENSE-LGPL" }
```

Comment:
Is everything available under LGPL now?

Reply:
I would expect that this license expression is valid for everything distributed with the python package (code + data).

In IMAS-Data-Dictionaries I've put it like this: https://github.com/iterorganization/IMAS-Data-Dictionaries/blob/main/pyproject.toml#L9-L10

```text
license = "LGPL-3.0+ AND CC-BY-ND-4.0"
license-files = ["LICENSE-LGPL", "LICENSE-CC-BY-ND"]
```